test/lib/organisations_fetcher_test.rb

```diff
@@ -113,4 +113,19 @@ class OrganisationsFetcherTest < ActiveSupport::TestCase
       OrganisationsFetcher.new.call
     end
   end
+
+  {
+    "animal-and-plant-health-agency" => "department-for-environment-food-rural-affairs",
```

Comment:
I agree with Alan that it would be nicer to iterate over `MANUAL_PARENT_FIXES` rather than repeat the config in two places.

I think we could either move `MANUAL_PARENT_FIXES` to the top of the class so that it's public (so long as the `.freeze` is on it, it should be safe to do that I think), or we could use [`instance_variable_set`](https://apidock.com/ruby/Object/instance_variable_set) to mock the `MANUAL_PARENT_FIXES` variable with some foobar values. Both would be preferable to duplicating the config IMO.

Reply:
Since you both think we should just iterate over `MANUAL_PARENT_FIXES`, I've amended the commit to do that!